rpcserver.go

```diff
@@ -456,6 +460,83 @@ func determineFeePerKw(feeEstimator lnwallet.FeeEstimator, targetConf int32,
 	}
 }
 
+func (r *rpcServer) ListUnspent(ctx context.Context,
```

Comment:
Missing godoc.

Reply:
Fixed in 60ae2afc236c6eb3d1095cca9dfed5d9760a23b4